src/erlastic_search.erl

```diff
@@ -648,6 +652,34 @@ aliases(Body) ->
 aliases(Params, Body) ->
     erls_resource:post(Params, filename:join([<<"_aliases">>]), [], [], erls_json:encode(Body), Params#erls_params.http_client_options).
 
+%%--------------------------------------------------------------------
+%% @doc
+%% Takes a list of operations of the type {Operation, Index, Type, Id, Json}
+%% Performs these operations using _bulk endpoint.
+%% The possible operations are index, create, delete and update.
+%% (see the doc at https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-bulk.html)
+%% @end
+%%--------------------------------------------------------------------
+-spec bulk_operation(operations()) -> {ok, list()} | {error, any()}.
```

Comment:
Also, would make more sense to have an `operation()` spec and then just have this fun take a `[operation()]`